index.js

```diff
@@ -54,6 +53,13 @@ async function startServer(config, logger) {
     });
 }
 
+function applyEntryPoints(app, currentDirectory, entryPoints = []) {
+    entryPoints.forEach(entry => {
+        const urlPath = entry.route || '/';
+        app.use(urlPath, express.static(path.normalize(`${currentDirectory}/${entry.entry}`)));
+    });
+}
+
 async function run(config, {logger}) {
     if (config.currentDirectory && config.entryPoints && config.entryPoints.length) {
```

Comment:
Still checking for entryPoints in the run(). Also, I think `currentDirectory` is only needed if entryPoints are being used.

Reply:
Oops -- bad copy/paste job from my test file. Should be fixed now